src/ripple/nodestore/impl/Shard.h

```diff
@@ -22,6 +22,7 @@
 
 #include <ripple/app/ledger/Ledger.h>
 #include <ripple/basics/BasicConfig.h>
+#include <ripple/basics/ProgressDisplay.h>
```

Comment:
This `#include` can be moved to Shard.cpp instead of here to help reduce the dependency tree.

Reply:
Removed